config/pipeline.yaml

```diff
@@ -1664,6 +1676,9 @@ scheduler:
 
   - job: kbuild-gcc-12-riscv-build-only
     <<: *build-k8s-all
+    rules:
```

Comment:
Actually I assume we also want to apply this rule to `kbuild-gcc-12-riscv` as well, right? This job [got triggered](https://staging.kernelci.org:9000/viewer?node_id=66b20bee2fbd8114fc0b7438) on `efi` (worked fine to exclude ARC/MIPS builds though, and `kbuild-gcc-12-riscv-build-only` as well)

Reply:
Done